Dockerfile

```diff
@@ -94,11 +94,14 @@ CMD /start-cadence.sh
 # All-in-one Cadence server
 FROM cadence-server AS cadence-auto-setup
 
-RUN apk add --update --no-cache ca-certificates py-pip mysql-client
-RUN pip install cqlsh
+RUN apk add --update --no-cache ca-certificates py-pip mysql-client python3
```

Comment:
`py-pip` depends on `python`(2): https://pkgs.alpinelinux.org/package/v3.3/main/x86/py-pip -> https://pkgs.alpinelinux.org/package/v3.3/main/x86/python
and afaict mysql-client doesn't need python at all.

so we can probably just change this to `py3-pip mysql-client`

Reply:
sounds good